docs/source/user-guide/concepts/environments.rst

```diff
@@ -52,4 +52,85 @@ environment:
 |
 
 Other conda environments usually contain the same subdirectories
-as the default environment.
\ No newline at end of file
+as the default environment.
+
+Virtual environments
+====================
+
+A virtual environment is a tool that helps to
+keep dependencies required by different projects
+separate by creating isolated Python virtual environments
+for them.
+
+Users can select a Python virtual environment, such as
+Pipenv or Poetry, or a conda virtual
+environment. Some characteristics of the two:
+
+.. list-table::
+   :widths: 20 40 40
+   :header-rows: 1
+
+   * - 
+     - Python virtual environment
+     - Conda virtual environment
+   * - **Libraries**
+     - Statically link, vendor libraries in wheels,
+       or use apt/yum/brew/etc.
+     - Install system-level libraries as conda dependencies
+   * - **System**
+     - Depend on base system install of Python
+     - Python is independent from system
+   * - **Extending environment**
+     - Extend environment with pip
+     - Extended environment with conda or pip
+   * - **Non-Python dependencies**
+     -
+     - Manages non-Python dependencies (R, Perl,
+       arbitrary executables)
+   * - **Tracking dependencies**
+     -
+     - Tracks binary dependencies explicitly
+
+|
+
+Why use Python virtual environments
+===================================
+- You prefer their workflow or spec formats
+- You prefer to use the system Python and libraries
+- You prefer packages that come more directly from the project maintainers
```

Comment:
```suggestion
- Ryour project maintainers only publish to PyPI, and you prefer packages that come more directly from the project maintainers, rather than someone else providing builds based on the same code
```